src/pages/nft/claim/index.vue

```diff
@@ -737,6 +740,9 @@ export default {
       if (newValue) {
         this.$router.push({ query: { ...this.$route.query, state: newValue } });
       }
+      if (newValue === NFT_CLAIM_STATE.LOGIN && this.claimingAddress) {
+        this.state = NFT_CLAIM_STATE.ID_CONFIRMATION;
+      }
```

Comment:
This should be put before `$router.push()` if check?

Reply:
[🎨 Set state before pushing query](https://github.com/likecoin/liker-land/pull/1616/commits/98883bf5e71c03370eb450499af192294ebee1fd)